gridstatus/gs_logging.py

```diff
@@ -1,3 +1,24 @@
+import logging
+
+
+def setup_gs_etl_logger():
```

Comment:
Function name?

Reply:
made a ticket to address the logging separately than this PR. https://www.notion.so/gridstatus/Update-gridstatus-to-use-detailed-logger-11be835f42aa808f82c3d16d7365abd1?pvs=4

Reply:
Sorry, I meant the function name should be updated for this repository.

Reply:
oh yeah, got it, done.